src/MongoDB/Manager.c

```diff
@@ -29,6 +29,8 @@
 
 #define PHONGO_MANAGER_URI_DEFAULT "mongodb://127.0.0.1/"
 
+ZEND_EXTERN_MODULE_GLOBALS(mongodb)
```

Comment:
It was definitely needed when I wrote the code. If any code in this file uses MONGODB_G, it needs to be there. Seems that we removed all of that code though.